pkg/test/framework/components/stackdriver/kube.go

```diff
@@ -174,6 +174,8 @@ func (c *kubeComponent) ListLogEntries() ([]*loggingpb.LogEntry, error) {
 	for _, l := range r.Entries {
 		// Remove fields that do not need verification
 		l.Timestamp = nil
+		l.Trace = ""
```

Comment:
I agree that it is not necessary to verify trace and span id, but we do verify log entry before this change. Wondering what is in this change that makes this issue surface?

Reply:
I think it might because accesslog verify trace and edge with validateTrace and ValidateEdge function. only validate log are using this ListLogEntries() func.

Reply:
Actually no I think it might because of trace sampling. Anyway, I think it is right to not verify trace related labels.